ansible/roles/dev-desktop/files/team_login/src/main.rs

```diff
@@ -0,0 +1,105 @@
+const TEAM_URL: &str = "https://team-api.infra.rust-lang.org/v1/teams/all.json";
```

Comment:
I'd add a new `dev-desktop = true` permission rather than just allowing the `all` team. Mostly because that'd allow to manually set `dev-desktop = false` on any user or team if problems arise.